### PR TITLE
traverse

### PR DESCRIPTION
**Pull request by PuruVJ**

### CODE REVIEW

**Comment by ronanru** [comment]
Please run `npm run format`

**Comment by PuruVJ** [comment]
Done

File: test/fixtures/traverse/case-1/result.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// THESE ARE FOR ESM IMPORTS
+import assert from 'assert';
+import traverse from 'neotraverse/legacy';
```
Comment (thepassle) [inline test/fixtures/traverse/case-1/result.js:3]: @PuruVJ can we add a test case for the `require` case as well?

Comment (PuruVJ) [reply test/fixtures/traverse/case-1/result.js:3]: Done

**Pull request merged**